gradle.properties

```diff
@@ -3,13 +3,12 @@ projectGroup=io.micronaut.grpc
 micronautDocsVersion=2.0.0
 micronautVersion=3.1.3
 micronautTestVersion=3.0.3
+
 micronautGradlePluginVersion=2.0.8
```

Comment:
what is this for? `micronautGradlePluginVersion`?

Reply:
They are used here:

https://github.com/micronaut-projects/micronaut-grpc/blob/8b5f55876bcb12ea50e2c9ea53ef2c73b616cd25/settings.gradle#L3-L4

We can't use catalogs for plugin versions, and I decided to leave these in the properties file as I guessed that would be the first place people would look for them